dracut/30ignition/ignition-fetch.service

```diff
@@ -6,6 +6,8 @@ DefaultDependencies=false
 Before=ignition-complete.target
 After=basic.target
 ConditionPathExists=/run/ignition/neednet
+# Don't run if the `fetch` stage successfully fetched a config
```

Comment:
s/fetch/fetch-offline/ or did I misunderstand the meaning of this?

Reply:
Good catch, fixed!